src/main/java/seedu/address/model/person/Person.java

```diff
@@ -118,6 +132,12 @@ public String toString() {
             builder.append("; Tags: ");
             tags.forEach(builder::append);
         }
+
+        List<Note> notes = getNotes();
+        if (!notes.isEmpty()) {
+            builder.append("; Note: ");
+            notes.forEach(builder::append);
```

Comment:
Incorporate the index (+1) of each note before appending it, so as to have the notes be a "list" for seamless management of notes.

Reply:
Thank you. I've updated the display to the indexed way.